packages/salesforce/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @openfn/language-salesforce
 
+## 4.5.2
+
+### Patch Changes
+
+- 2006e88: await `job.close()`
```

Comment:
Maybe we can say "fix an issue with bulk jobs not closing"

Also the PR needs to come out of draft so I can merge.

Thanks!